docs/available_checks.md

```diff
@@ -0,0 +1,69 @@
+# Available Checks
+This document is a list of tables with a description and link to documentation for each check.  Each table is organized by check type.
+
+## Areas
+| Check Name | Check Description |
+| :--------- | :---------------- |
+| AreasWithHighwayTagCheck | The purpose of this check is to identify Areas attributed with highway tags. |
+| [PoolSizeCheck](docs/tutorials/tutorial1-PoolSizeCheck.md) | The purpose of this check is to identify pools that are larger than 5,000,000 square meters or smaller than 5 square meters.  This check was created to be used as a tutorial for developing new checks. |
```

Comment:
Since the Available Checks is already in the docs directory and all the checks .md files are also in the docs directory, we can omit docs/ from the url. For example, (docs/tutorials/tutorial1-PoolSizeCheck.md) will be (tutorials/tutorial1-PoolSizeCheck.md)